nodeup/pkg/model/ntp.go

```diff
@@ -120,12 +127,16 @@ func (b *NTPBuilder) Build(c *fi.ModelBuilderContext) error {
 	return nil
 }
 
-func UpdateNtpIP(ip string, ntp string) ([]byte, error) {
+// UpdateNtpIP takes a ip and a NtpDaemon and will comment out 
+// the default server or pool values and append the correct cloud 
+// ip to the ntp config file
+func UpdateNtpIP(ip string, ntpd NtpDaemon) ([]byte, error) {
```

Comment:
The parameter is shadowing the constant of the same name. They should have different names.

Reply:
Thanks @johngmyers would you be happy if I just used ```func UpdateNtpIP(ip string, n NtpDaemon)``` as its clear that it is now by the type?

Reply:
That is acceptable, though I would suggest `daemon`.

Reply:
I have switched the param to daemon.  Thanks @johngmyers